### ISSUE TITLE
UI initialization fails

### ISSUE THREAD

**Issue by askingcat** [subshare/subshare#15]
The GUI client failed to start while trying to initialize the backup dialog on a Windows 7 system. Since I already ran subshare successfully on other computers with Windows 7, I assume that actually one of the "special" system configurations mentioned below might be important. This issue can be reproduced on the system (occurs on every start, independently of the home/config folder of subshare).
[subshare.client.log.txt](https://github.com/subshare/subshare/files/410341/subshare.client.log.txt)

Used version: 0.9.2
OS: Windows 7
Noteworthy: 
    * home directory is mounted/on a remote disk
    * several partitions are network shares

**Comment by nlmarco** [comment]
Cannot reproduce.

I tried it in a Windows 7 VM. Even mounting a partition to a directory (instead of a drive letter) didn't cause any trouble.

I therefore removed this sanity check (it is nothing else and code should work fine without it).

The problem btw. occurred (according to the stack trace) in the following constellation:

```
for (File root : File.listRoots()) {
    if (! root.isDirectory())
        throw new IllegalStateException("root is not a directory: " + root);
}
```

You might want to try the above few lines in your environment.

Also I added a WARN log to `RootFileTreeItem.loadChildren()`, if it encounters this strange situation. But it still continues to create the `DirectoryFileTreeItem`.
